edge-modules/TwinTester/Settings.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TwinTester
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Configuration;
+    using Newtonsoft.Json;
+
+    public class Settings
+    {
+        static readonly Lazy<Settings> DefaultSettings = new Lazy<Settings>(
+            () =>
+            {
+                IConfiguration configuration = new ConfigurationBuilder()
+                    .SetBasePath(Directory.GetCurrentDirectory())
+                    .AddJsonFile("config/settings.json", optional: true)
+                    .AddEnvironmentVariables()
+                    .Build();
+
+                return new Settings(
+                    configuration.GetValue<string>("IOTEDGE_DEVICEID", string.Empty),
+                    configuration.GetValue<string>("IOTEDGE_MODULEID", string.Empty),
+                    configuration.GetValue<int>("TwinUpdateSize", 1),
```

Comment:
```suggestion
                    configuration.GetValue<uint>("TwinUpdateSize", 1),
```

Reply:
The preconditions will catch negative numbers

Reply:
I feel preconditions is cleaner than having to cast to int everywhere we want to use the field.

Reply:
I have preconditions set up to throw an exception in this case. Assuming that is fine?

Reply:
Once use uint and negative value is provided, it will throw cast exception.